tests/test_scheduler.py

```diff
@@ -714,7 +714,7 @@ def test_full_loop_no_noise(self):
         result_sum = torch.sum(torch.abs(sample))
         result_mean = torch.mean(torch.abs(sample))
 
-        assert abs(result_sum.item() - 14379591680.0) < 1e-2
+        assert abs(result_sum.item() - 14379589632.0) < 1e-2
```

Comment:
Circle CI seemed to be happy with `14379591680.0`: https://github.com/huggingface/diffusers/runs/8068605613?check_suite_focus=true#step:6:71 So not sure about this change

=> maybe we should increase the tolerance here?

Reply:
yes perhaps that is what i was thinking that too so perhaps abs < 2.5 e3 or so